bundles/binding/org.openhab.binding.wr3223/src/main/java/org/openhab/binding/wr3223/WR3223CommandType.java

```diff
@@ -0,0 +1,798 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.wr3223;
+
+import org.openhab.binding.wr3223.internal.WR3223Commands;
+import org.openhab.core.items.Item;
+import org.openhab.core.library.items.ContactItem;
+import org.openhab.core.library.items.NumberItem;
+import org.openhab.core.library.items.SwitchItem;
+
+/**
+ * Represents all valid commands which could be processed by this binding
+ *
+ * @author Michael Fraefel
+ * @since 1.8.0
+ */
+public enum WR3223CommandType {
+
+    /** T1 (de: Verdampfertemperatur) */
+    TEMPERATURE_EVAPORATOR {
+        {
+            command = "temperature_evaporator";
+            itemClass = NumberItem.class;
+            wr3223Command = WR3223Commands.T1;
+        }
+    },
+
+    /** T2 (de: Kondensatortemperatur) */
+    TEMPERATURE_CONDENSER {
+        {
+            command = "temperature_condenser";
+            itemClass = NumberItem.class;
+            wr3223Command = WR3223Commands.T2;
+        }
+    },
+
+    /** T3 (de: Aussentemperatur) */
+    TEMPERATURE_OUTSIDE {
+        {
+            command = "temperature_outside";
+            itemClass = NumberItem.class;
+            wr3223Command = WR3223Commands.T3;
+        }
+    },
+
+    /** T4 (de: Ablufttemperatur (Raumtemperatur)) */
+    TEMPERATURE_OUTGOING_AIR {
+        {
+            command = "temperature_outgoing_air";
+            itemClass = NumberItem.class;
+            wr3223Command = WR3223Commands.T4;
+        }
+    },
+
+    /** T5 (de: Temperatur nach Wärmetauscher (Fortluft)) */
+    TEMPERATURE_AFTER_HEAT_EXCHANGER {
+        {
+            command = "temperature_after_heat_exchanger";
+            itemClass = NumberItem.class;
+            wr3223Command = WR3223Commands.T5;
+        }
+    },
+
+    /** T6 (de: Zulufttemperatur) */
+    TEMPERATURE_SUPPLY_AIR {
+        {
+            command = "temperature_supply_air";
+            itemClass = NumberItem.class;
+            wr3223Command = WR3223Commands.T6;
+        }
+    },
+
+    /** T7 (de: Temperatur nach Solevorerwärmung) */
+    TEMPERATURE_AFTER_BRINE_PREHEATING {
+        {
+            command = "temperature_after_brine_preheating";
+            itemClass = NumberItem.class;
+            wr3223Command = WR3223Commands.T7;
+        }
+    },
+
+    /** T8 (de: Temperatur nach Wärmetauscher) */
+    TEMPERATURE_AFTER_PREHEATING {
+        {
+            command = "temperature_after_heat_exchanger";
```

Comment:
TEMPERATURE_AFTER_PREHEATING and TEMPERATURE_AFTER_HEAT_EXCHANGER  have the same command "temperature_after_heat_exchanger" asigned. I think this will result in TEMPERATURE_AFTER_PREHEATING not to work. Or am I missing something?